CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.17.3
+
+
+### Bug Fixes
+
+  - Ensure that Telemetry heartbeats are not skipped for forked processes, as doing so could result in the dependency list being lost over time.
```

Comment:
Could you add appropriate product name in front of this entry? is it tracer or asm?